<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ir2Usb</class>
 <widget class="QWidget" name="Ir2Usb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ir2Usb</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);
font: 75 10pt &quot;Purisa&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="registerTab">
    <attribute name="title">
     <string>Register</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <widget class="QPushButton" name="writeButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>99</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Content</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="writeAddText">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 12pt &quot;Purisa&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="writeDataText">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 12pt &quot;Purisa&quot;;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>-1</number>
     </property>
     <widget class="QPushButton" name="readButoon">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>99</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Content</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="readAddText">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 12pt &quot;Purisa&quot;;
texe-align: center</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="readDataText">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 12pt &quot;Purisa&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="updateTab">
    <attribute name="title">
     <string>Update</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
     </property>
     <property name="text">
      <string>Update type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="updateChoice">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
font: 11pt &quot;Purisa&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>FW update</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SW update</string>
      </property>
     </item>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>621</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Current FW version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Current SW version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="fwCurrentLab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="swCurrentLab">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>unknown</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refCurrVerButt">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="fileNameText">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>70</y>
       <width>601</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
font: 75 12pt &quot;Purisa&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startUpdButt">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
font: 75 11pt &quot;Purisa&quot;;</string>
     </property>
     <property name="text">
      <string>Start Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="aboutTab">
    <attribute name="title">
     <string>About</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
